tests/c/CMakeLists.txt

```diff
@@ -21,29 +21,9 @@ if(BUILD_TESTS)
 
     target_include_directories(unittests PRIVATE ${libcsound_private_include_dirs})
     target_include_directories(unittests PUBLIC ${libcsound_public_include_dirs})
-
-    # Required because we're compiling the tests written in C as C++
-    set(CMAKE_CXX_FLAGS "-fpermissive -Wwrite-strings -std=c++0x")
```

Comment:
- `-fpermissive` isn't necessary anymore based on CI.
- Fine to leave in `write-strings` warnings
- gtest requires C++14